packages/atomic/src/components/common/generated-answer/styles/generated-answer.pcss

```diff
@@ -20,7 +20,19 @@
   }
 
   [part='rephrase-button'] {
-    @apply rephrase-with-text;
+    @apply rephrase-with-text flex-auto justify-center;
```

Comment:
`flex-auto justify-center` can be set as `style` directly in JSX

Reply:
@olamothe this is specific for mobile layout that s why they are declared here